edgelet/hsm-sys/azure-iot-hsm-c/src/edge_pki_openssl.c

```diff
@@ -795,6 +799,78 @@ static int set_basic_constraints(X509 *x509_cert, CERTIFICATE_TYPE cert_type, in
     return result;
 }
 
+static int add_ext(X509 *x509_cert, int nid, char *value, const char* nid_diagnostic)
```

Comment:
Are we using this call yet? Or is this in prep for the identity cert work?

Reply:
Yes we are starting with this PR and this is also prep for ID cert coming up next.

Reply:
This also solves the problem where the quick start certs could not be used for E2E TG tests.

Reply:
ok cool